modules/monitoring-setting-up-alert-routing-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-alerts-for-user-defined-projects.adoc
+
+:_content-type: CONCEPT
+[id="setting-up-alert-routing-for-user-defined-projects_{context}"]
+= Setting up alert routing for user-defined projects
+
+[role="_abstract"]
+As a cluster administrator, you can set up alert routing for user-defined projects. 
+After doing so, you can allow users to configure alert routing for their user-defined projects using either the default Alertmanager instance or the optional Alertmanager instance dedicated only to user workload monitoring, if enabled. 
+Users can then create and configure user-defined alert routing by creating or editing the `AlertmanagerConfig` objects for their user-defined projects without the help of an administrator.
+
+After a user has defined alert routing for a user-defined project, user-defined alerts are routed as follows:
+
+* To the `alertmanager-main` pods in the `openshift-monitoring` namespace if using the default platform Alertmanager instance.
+
+* To the `alertmanager-user-workload` pods in the `openshift-user-workload-monitoring` namespace if you have enabled a separate instance of Alertmanager for user-defined projects. 
+
+Note the following limitations and features of alert routing for user-defined projects:
+
+* For user-defined alerting rules, user-defined routing is scoped to the namespace in which the resource is defined.
+For example, a routing configuration in namespace `ns1` only applies to `PrometheusRules` resources in the same namespace.
+
+* The Cluster Monitoring Operator (CMO) does not deploy a second Alertmanager service dedicated to user-defined alerts.
```

Comment:
I'm not too sure about the meaning of this sentence.

Reply:
Deleting this sentence. I'm not sure why it was there from the previous release.